### ISSUE TITLE
Finish DtNonAsciiTests package and submit to CRAN

### ISSUE THREAD

**Issue by mattdowle** [Rdatatable/data.table#1343]
Investigate why the change in R-devel on 18 Sep 2015 caused 2 of data.table's non-ASCII tests (1092 & 1093) to fail on R-devel on Windows. data.table v1.9.4 on CRAN was very stable in 'OK' state for nearly a year before this fail.  Decided it's good to have non-ASCII tests in a separate package just to take a belt-and-braces approach.
See https://developer.r-project.org/blosxom.cgi/R-devel/NEWS
The change to match() is most likely I'd guess.

**Comment by jangorecki** [comment]
[Speedup for match(scalar, table) - from Pete Haverty`s PR#16491 ](https://github.com/wch/r-source/commit/b23076eda0957292f3fb8e945e4af2fd07fb8bec)
Another potential place, a change related to encoding on that time [Fix for PR#16264, mishandling of UTF-8 strings by grep() in multibyte locale](https://github.com/wch/r-source/commit/674617d04d7b91a20cfd5a97c1bbb74d61f3fef9).

Any reason why you want to use testthat?

**Comment by jangorecki** [comment]
@mattdowle can we remove https://github.com/Rdatatable/DtNonAsciiTests and close this issue?

**Comment by mattdowle** [comment]
@jangorecki Good idea.  But don't we need to move the tests in DtNonAscii back in to the main project first, tests 1088-1093 for example?   I just 'archived' `www` as a first step.
